portfolio/src/test/java/com/google/sps/NameTest.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
```

Comment:
the package should generally match the file path for the java class. so this file needs to be moved into `portfolio/src/test/java/com/google/sps/servlets` directory.